lib/time/src/index.js

```diff
@@ -5,9 +5,12 @@ var timeFx = function(fx) {
 }
 
 export var timeout = timeFx(function(dispatch, props) {
-  setTimeout(function() {
+  const id = setTimeout(function() {
```

Comment:
```suggestion
  var id = setTimeout(function() {
```

Reply:
Ough, thanks. Forgot that we are strict es5 :)